modules/reference/pages/command/featureUtility-installServerFeatures.adoc

```diff
@@ -24,6 +24,13 @@ Install the server features for the `myserver` server:
 featureUtility installServerFeatures myserver
 ----
 
+Install the custom user server features for the `myserver` server from a Maven repository:
+
```

Comment:
Install the features for the  the `myserver` server, which includes custom user features: